pybind11_protobuf/tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# ============================================================================
+# tests
+# ============================================================================
+
```

Comment:
This doesn't look like it's ever getting invoked

Reply:
Yes - the cmake build for tests is not yet complete and integrated. Can look to add if required for this PR.

Reply:
Could we just omit this file in this PR? Would we lose something if we did that?

Reply:
Yes can remove. No we won't lose anything.